### PR TITLE
Add libmsym check for cmake

### PR DESCRIPTION
**Pull request by serval2412**

### CODE REVIEW

File: avogadro/core/CMakeLists.txt [hunk 1]
```diff
@@ -8,6 +8,9 @@ if(USE_LIBSPG)
   include_directories(SYSTEM ${SPGLIB_INCLUDE_DIR})
 endif()
 
+find_package(Libmsym REQUIRED)
```
Comment (ghutchis) [inline avogadro/core/CMakeLists.txt]: No, it's not required - from `avogadrolibs/CMakeLists.txt`
```
option(USE_LIBMSYM "Enable optional features using libmsym" ON)
```

**Comment by ghutchis** [comment]
This is fine, but the call should only happen if USE_LIBMSYM is set.

**Comment by serval2412** [comment]
Thank you ghutchis for your feedback, I changed avogadro/core/CMakeLists.txt as requested

**Comment by serval2412** [comment]
Is there something I can do so the patch can be merged?
I noticed AppVeyor build failed but don't understand what's the relation with the patch.

**Comment by cryos** [comment]
It may be related, it looks like the crucial line was clipped though:
`CUSTOMBUILD : CMake error : The following variables are used in this project, but they are set to NOTFOUND.`
I will try and take a look at this error, and confirm whether or not it is related.

**Comment by cryos** [comment]
You should try to develop on named branches, I just pushed #252 which just cherry picks your final commit, and tested that. Looks good here - I am closing this PR, and merging the other. Thanks for your contribution.

**Pull request closed without merging**